global.R

```diff
@@ -0,0 +1,43 @@
+###### MEtaInsight ######
```

Comment:
Can you change this to MetaInsight please - the extra capital E has been bugging me for months!

Reply:
Done 😁